src/applications/personalization/rated-disabilities/sass/rated-disabilities.scss

```diff
@@ -10,13 +10,37 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
 .rated-disability-item-container {
```

Comment:
Im gonna add a bunch of comments in this file since we have utility classes that can be used to accomplish almost everything in this file.